addons/mail/static/src/js/activity.js

```diff
@@ -681,6 +682,7 @@ var KanbanActivity = BasicActivity.extend({
      * @private
      */
     _render: function () {
+        this.renderElement()
```

Comment:
why this change?

Reply:
@mge-odoo when kanban view is not group By with any field and we are scheduling the new activity then whole kanban view will not be rendered so icon is not going to update instantly so we need to re-render the element by using renderElement() or we may change the icon explicitly by selector